### PR TITLE
Update figma changelog

### PR DESCRIPTION
**Pull request by jacoblogan**
_Description of changes:_
Update the changelog of the figma community file and added a page describing the process of converting existing TextFields to TextAreaFields within Figma.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/pages/console/uibuilder/figmatocode.mdx [hunk 1]
```diff
@@ -62,6 +61,11 @@ import { ComponentA, ComponentB }  from './ui-components';
 
 ### Figma file changelog
 
+#### 1.2.0: July 25 2022
+
+* Add TextAreaField primitive component. For additional information on converting existing TextFields to TextAreaFields, see [here](/console/uibuilder/textfieldtoarea/).
+* Update supported Amplify UI React version to 3.1.0.
+
```
Comment (hein-j) [inline src/pages/console/uibuilder/figmatocode.mdx:68]: Maybe out of scope here, but are we missing changelogs for 1.1.0?

File: src/pages/console/uibuilder/textfieldtoarea.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Convert TextField to TextAreaField in Figma`,
+  description: `How to convert existing TextField primitive components to TextAreaField primitive components`
+};
+
+After [upgrading](/console/uibuilder/figmatocode/#upgrading-figma-files) to the current Figma community file. You can update any existing TextFields into TextAreaFields by selecting the desired component and changing the instance of the TextField component to the TextAreaField component.
```
Comment (hein-j) [inline src/pages/console/uibuilder/textfieldtoarea.mdx:6]: Copy change suggestion --

After upgrading to the current Figma community file, you can change any existing TextFields into TextAreaFields by selecting the desired TextField and choosing TextAreaFields under the Swap Instance dropdown.

Comment (jacoblogan) [reply src/pages/console/uibuilder/textfieldtoarea.mdx]: Thanks, updated.

**Review by hein-j — APPROVED** [review]
Thanks for updating the copy

File: src/pages/console/uibuilder/figmatocode.mdx (hunk 1 above)
Comment (jacoblogan) [reply src/pages/console/uibuilder/figmatocode.mdx:68]: Added a backlog task to update this changelog

**Review by renebrandel — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**